docs/source/teams/installation.rst

```diff
@@ -223,6 +223,32 @@ You can do this in any of the following ways:
 `boto3 <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials>`_
 python library.
 
+For convenience, below are the environment variables that can be set to
+use some of the AWS-supported credentialing methods. This list is subject to
+change; please see the `boto3` link above for the up-to-date reference.
+
+.. code-block:: bash
+
+    # Access Key
```

Comment:
Can you put these in separate `.. code-block:: bash` as done in the Azure section just to help clarify that these are distinct options (ie you don't have to provide all of them)?